Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingTracingFragment.kt

```diff
@@ -76,12 +70,13 @@ class OnboardingTracingFragment : Fragment(R.layout.fragment_onboarding_tracing)
         binding.onboardingTracingContainer.sendAccessibilityEvent(AccessibilityEvent.TYPE_ANNOUNCEMENT)
     }
 
+    // TODO update deprecated fun with registerForActivityResult() method
```

Comment:
it is reported by sonar https://sonarcloud.io/project/issues?id=corona-warn-app_cwa-app-android&pullRequest=5597&resolved=false&types=CODE_SMELL

Reply:
Ok I'll remove the comment and I'll try to open a task in Jira.

Reply:
Sounds good, I'm not sure if that is possible with ENS dialogs. I already reported an issue to Google some time ago ,but no updates yet https://issuetracker.google.com/issues/180199984